test-buildtool/src/test/groovy/io/micronaut/starter/core/test/buildTool/MavenPackageSpec.groovy

```diff
@@ -58,7 +58,7 @@ class MavenPackageSpec extends CommandSpec {
         String output = executeMaven( "package -Dpackaging=docker-native -Pgraalvm", 30)
 
         then:
-        output.contains("Using BASE_IMAGE: ghcr.io/graalvm/native-image:ol7-java11-22.1.0")
+        output.contains("Using BASE_IMAGE: ghcr.io/graalvm/native-image:ol7-java11-22.2.0")
```

Comment:
This will fail, Maven plugin isn't updated yet.

Reply:
Actually, I stand corrected. GraalVM version comes from the BOM and you updated here: https://github.com/micronaut-projects/micronaut-core/pull/7742

It's just that the new snapshot build hasn't finished yet.

So this will eventually work.